virttest/qemu_devices/qdevices.py

```diff
@@ -170,7 +170,7 @@ def __str__(self):
 
     def __eq__(self, dev2, dynamic=True):
         """ :return: True when devs are similar, False when different. """
-        if not isinstance(dev2, QBaseDevice):
+        if not isinstance(dev2, type(self)):
```

Comment:
Good catch, but I'm afraid we have to run some regression tests to check whether this change would break some upper workflows, thanks.

Reply:
I run the gating test loop and there is no regression error I found, all tests are passed.
Thanks.